Create using-breakout-rooms.md (#6337)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/using-breakout-rooms.md

---

> title: Using Breakout Rooms in Teams

Reviewer: ```suggestion
title: Using breakout rooms in Teams
```
```suggestion
title: Using Breakout Rooms in Teams
```

> title: Using Breakout Rooms in Teams

Reviewer: ```suggestion
title: Using breakout rooms in Teams
```

ms.topic: article
ms.service: msteams
ms.collection: 
  - M365-collaboration
appliesto: 
  - Microsoft Teams

> description: End Users can learn how to use Breakout Rooms in Microsoft Teams

Reviewer: ```suggestion
description: End users can learn how to use breakout rooms in Microsoft Teams
```

---

> # Using Breakout Rooms in Microsoft Teams

Reviewer: ```suggestion
# Using breakout rooms in Microsoft Teams
```
```suggestion
# Using Breakout Rooms in Microsoft Teams
```

As a meeting organizer, you can create new breakout rooms at the start of a scheduled meeting.

## Create breakout rooms

> Use these steps to create a breakout room once a meeting begins.

Reviewer: ```suggestion
Use these steps to create a breakout room once a meeting begins:
```

> 1. In theTeams desktop client, select the Breakout rooms icon on the control bar.

Reviewer: ```suggestion
1. In the Teams desktop client, select the breakout rooms icon on the control bar.
```

> 2. Use the dropdown menu to select the number of rooms you would like. You can create up to 50 breakout rooms in a single meeting.

Reviewer: ```suggestion
2. Use the dropdown menu to select the number of rooms you would like to create. You can create up to 50 breakout rooms in a single meeting.
```

3. Choose the breakout room option you prefer:

>     - **Automatically:** Teams divides participants equally between the rooms.

Reviewer: ```suggestion
    - **Automatically**: Teams divides participants equally between the rooms.
```

>     - **Manually:** You can choose which room each participant is placed in.

Reviewer: ```suggestion
    - **Manually**: You can choose which room each participant is placed in.
```

> 4. Select **Create Rooms**. The Breakout Rooms pane will open.

Reviewer: ```suggestion
4. Select **Create Rooms**. The breakout rooms pane will open.
```

## Start breakout rooms

> When you are ready to start your breakout sessions, select **Start rooms.**

Reviewer: ```suggestion
When you are ready to start your breakout sessions, select **Start rooms**.
```

You will see participants leaving the main meeting and joining their breakout rooms.

> If a participant leaves their breakout room the organizer can move the participant back to the breakout room by selecting the ellipses […] next to the participant’s name and select Ask to join.

Reviewer: ```suggestion
If a participant leaves their breakout room, the organizer can move the participant back to the breakout room by selecting the ellipses **[…]** next to the participant’s name and select **Ask to join**.
```

> If a participant leaves their breakout room, the organizer can move the participant back to the breakout room by selecting the ellipses **[…]** next to the participant’s name and select **Ask to join**.

Reviewer: ```suggestion
If a participant leaves their breakout room, the organizer can move the participant back to the breakout room by selecting the ellipses **[…]** next to the participant’s name and then selecting **Ask to join**.
```

> If a participant joins the main meeting late after breakout rooms have opened, the organizer can add the participant to a breakout room. If the participant misses the notification to join the room, select Ask to join.

Reviewer: ```suggestion
If a participant joins the main meeting late after breakout rooms have opened, the organizer can add the participant to a breakout room. If the participant misses the notification to join the room, select **Ask to join**.
```

> ## Send Announcements

Reviewer: ```suggestion
## Send announcements
```

> The meeting organizer can send announcements to all participants while breakout rooms are open. Select the ellipses […] by the Breakout Rooms header.

Reviewer: ```suggestion
The meeting organizer can send announcements to all participants while breakout rooms are open. Select the ellipses **[…]** in the breakout rooms header.
```

> For example, remind participants to @mention the meeting organizer when they need help, or give them a fiveminute countdown before pulling them back into the main room.

Reviewer: ```suggestion
For example, remind participants to @mention the meeting organizer when they need help, or give them a five-minute countdown before pulling them back into the main room.
```

Participants receive the announcement in their breakout rooms’ chat.

> ## Close Rooms and Pull participants back to the main meeting

Reviewer: ```suggestion
## Close rooms and pull participants back to the main meeting
```

> Select Close rooms to pull all the participants back to the main room.

Reviewer: ```suggestion
Select **Close rooms** to pull all the participants back to the main room.
```

> Or select the ellipses […] for a specific breakout room, then select Close room.

Reviewer: ```suggestion
Alternatively, select the ellipses **[…]** for a specific breakout room, then select **Close room**.
```
